src/components/emoji-picker/EmojiPicker.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { EmojiPicker } from './EmojiPicker';
+
+export const Default = ({ ...args }) => {
+  return (
+    <EmojiPicker onSelect={(e) => console.log("Selected emoji: " + e)}>
```

Comment:
@steiningermona  maybe you can also use the Emoji component here as an example and a setState where you set the selected emoji and then display the emoji with the <Emoji ... component in the story